twoopstracker/twoops/serializers.py

```diff
@@ -94,7 +94,10 @@ def get_accounts(self, obj):
 
     def to_representation(self, instance):
         data = super().to_representation(instance)
-        data["accounts"] = self.get_accounts(instance)
+        if self.context.get("request", {}).GET.get("download"):
+            data["accounts"] = self.get_accounts(instance)
+        else:
+            del data["accounts"]
```

Comment:
```suggestion
            data.pop("accounts", None)
```

Just to avoid `KeyError` if any